src/test/java/bio/terra/common/BaseTest.java

```diff
@@ -100,6 +102,13 @@ public void assertStorageObjectData(StorageObjectData data) {
     assertEquals("value2", data.getMetadata().get("key2"));
   }
 
+  public void assertGCSEvent(GCSEvent data) {
+    // Check String deserialization
+    assertEquals("terra-kernel-k8s-testrunner-results", data.getBucket());
+    // Check OffsetDateTime deserialization
+    assertEquals("2021-07-07T22:57:14.257Z", data.getTimeCreated().toInstant());
```

Comment:
Curious - what's the criteria for choosing to assert on these two (out of many) GCSEvent fields?

Reply:
I simply picked the bucket name and a `Date` object for assertion out of economy instead of all the properties.